x-pack/plugins/security_solution/public/management/pages/blocklist/view/blocklist.tsx

```diff
@@ -48,52 +51,47 @@ const BLOCKLIST_PAGE_LABELS: ArtifactListPageProps['labels'] = {
   ),
   flyoutCreateSubmitSuccess: ({ name }) =>
     i18n.translate('xpack.securitySolution.blocklist.flyoutCreateSubmitSuccess', {
-      defaultMessage: '"{name}" has been added to your blocklist.', // FIXME: match this to design (needs count of items)
+      defaultMessage: '"{name}" has been added to your blocklist.',
       values: { name },
     }),
   flyoutEditSubmitSuccess: ({ name }) =>
     i18n.translate('xpack.securitySolution.blocklist.flyoutEditSubmitSuccess', {
       defaultMessage: '"{name}" has been updated.',
       values: { name },
     }),
-  flyoutDowngradedLicenseDocsInfo: () => {
-    return 'tbd...';
-    // FIXME: define docs link for license downgrade message. sample code below
-
-    // const { docLinks } = useKibana().services;
-    // return (
-    //   <FormattedMessage
-    //     id="some-id-1"
-    //     defaultMessage="For more information, see our {link}."
-    //     value={{
-    //       link: (
-    //         <EuiLink target="_blank" href={`${docLinks.links.securitySolution.eventFilters}`}>
-    //           {' '}
-    //           <FormattedMessage
-    //             id="dome-id-2"
-    //             defaultMessage="Event filters documentation"
-    //           />{' '}
-    //         </EuiLink>
-    //       ),
-    //     }}
-    //   />
-    // );
+  flyoutDowngradedLicenseDocsInfo: (
+    securitySolutionDocsLinks: DocLinks['securitySolution']
+  ): React.ReactNode => {
+    return (
+      <>
+        <FormattedMessage
+          id="xpack.securitySolution.blocklist.flyoutDowngradedLicenseDocsInfo"
+          defaultMessage="For more information, see our "
+        />
+        <EuiLink target="_blank" href={`${securitySolutionDocsLinks.blocklist}`}>
+          <FormattedMessage
+            id="xpack.securitySolution.blocklist.flyoutDowngradedLicenseDocsLink"
+            defaultMessage="Blocklist documentation"
+          />
```

Comment:
In the screenshot above, there's a period missing at the end of this sentence, and "blocklist documentation" should probably be lowercase since the feature isn't a proper name.

"For more information, see our blocklist documentation."